File: LLM-Translations/GPT-4/codenet/Python/Go/s923405856.go
package main

import (
	"fmt"
)

func main() {
	for {
		var n int
		fmt.Scan(&n)

		if n == 0 {
			break
		}

		res := -1111111111
		s := 0
		for i := 0; i < n; i++ {
			var a int
			fmt.Scan(&a)
			if s+a > a {
				s = s + a
			} else {
				s = a
			}
			if s > res {
				res = s
			}
		}

		fmt.Println(res)
	}
}

// 

File: LLM-Translations/GPT-4/codenet/Python/C/s923405856.c
#include <stdio.h>

int main() {
    int n, a, res, s;

    while (1) {
        scanf("%d", &n);

        if (n == 0) {
            break;
        }

        res = -1111111111;
        s = 0;
        for (int i = 0; i < n; i++) {
            scanf("%d", &a);
            s = (s + a > a) ? s + a : a;
            res = (s > res) ? s : res;
        }

        printf("%d\n", res);
    }

    return 0;
}
// 
